test/unit_tests/braket/schema_common/test_schema_base.py

```diff
@@ -62,3 +62,14 @@ def test_parse_raw_schema():
     )
     assert schema == BraketSchemaBase.parse_raw_schema(schema.json())
     assert isinstance(schema, TaskMetadata)
+
+
+def test_get_schema_class():
```

Comment:
Add a test for the negative case as well - make sure an undefined module will throw an error.